src/components/Separator/SeparatorStyles.js

```diff
@@ -0,0 +1,13 @@
+import { View, StyleSheet } from 'react-native'
+import colors from '../../utils/colors';
+
+export default StyleSheet.create({
+    separator: {
+      backgroundColor: colors.gray,
+      minHeight: StyleSheet.hairlineWidth,
```

Comment:
[Minor comment]

There is new Rn feature for hairline separator: https://facebook.github.io/react-native/docs/stylesheet#hairlinewidth

Probably we may use it instead of whole component Separator.